packages/hdwallet-tally/src/adapter.ts

```diff
@@ -0,0 +1,46 @@
+import * as core from "@shapeshiftoss/hdwallet-core";
+import { TallyHDWallet } from "./tally";
+import TallyOnboarding from "tallyho-onboarding";
+import detectEthereumProvider from "@metamask/detect-provider";
+
+export class TallyAdapter {
+  keyring: core.Keyring;
+
+  // wallet id to remove from the keyring when the active wallet changes
+  currentDeviceID?: string;
+
+  private constructor(keyring: core.Keyring) {
+    this.keyring = keyring;
+  }
+
+  public static useKeyring(keyring: core.Keyring) {
+    return new TallyAdapter(keyring);
+  }
+
+  public async initialize(): Promise<number> {
+    return Object.keys(this.keyring.wallets).length;
+  }
```

Comment:
Unused, remove as in #431

Reply:
Fixed in [0b0e667](https://github.com/shapeshift/hdwallet/pull/506/commits/0b0e667653a33fb7fb9ddd4905b81979b6cb1fa8)